app/controllers/coronavirus_form/offer_other_support_controller.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::OfferOtherSupportController < ApplicationController
+  include ActionView::Helpers::SanitizeHelper
+  include FieldValidationHelper
+
+  def show
+    render "coronavirus_form/#{PAGE}"
+  end
+
+  def submit
+    offer_other_support = sanitize(params[:offer_other_support]).presence
+
+    session[:offer_other_support] = offer_other_support
+
+    invalid_fields = validate_radio_field(
+      PAGE,
+      other: support_other,
+    )
+
+    if invalid_fields.any?
+      flash.now[:validation] = invalid_fields
+      render "coronavirus_form/#{PAGE}"
+    elsif session["check_answers_seen"]
+      redirect_to controller: "coronavirus_form/check_answers", action: "show"
+    elsif session[:offer_other_support] == I18n.t("coronavirus_form.offer_other_support.options.option_yes.label")
```

Comment:
It looks like this line would need to be updated as we're missing an `option_yes.label` for the section.

Reply:
great added that, thanks for the spot